css/owned-card.styl

```diff
@@ -58,8 +58,28 @@
       background: lighten(COBOLT_BLUE, 20%)
 
 .collection-card
+  -webkit-border-radius: 1em
+  border-radius: 1em
+
   .name
     margin-bottom: 1em
+    height: 2.9em
+    overflow: hidden
+    text-overflow: ellipsis
+    display: -webkit-box
+    -webkit-box-orient: vertical
+    -webkit-line-clamp: 2
```

Comment:
Nib will read in modern flexbox properties and output something as cross-browser compatible as possible. In general, manually included vendor prefixes should be a rare occurrence.

I've got a little brute-force string truncator component coming with the new home page, which might be useful here. https://github.com/zooniverse/Panoptes-Front-End/blob/new-signed-in-home-page/app/pages/home-for-user/string-truncator.jsx